plugins/catalog-backend/src/ingestion/processors/FileReaderProcessor.test.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FileReaderProcessor } from './FileReaderProcessor';
+import {
+  CatalogProcessorEntityResult,
+  CatalogProcessorErrorResult,
+  CatalogProcessorResult,
+} from './types';
+
+describe('FileReaderProcessor', () => {
+  const fixturesRoot =
+    'src/ingestion/processors/__fixtures__/fileReaderProcessor';
+
+  it('should load from file', async () => {
+    const processor = new FileReaderProcessor();
+    const spec = {
+      type: 'file',
+      target: `${fixturesRoot}/component.yaml`,
+    };
+
+    const generated = (await new Promise<CatalogProcessorResult>(emit =>
```

Comment:
Kinda neat!

Reply:
Stole that from the UrlReaderProcessor 😆